packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -24,8 +24,10 @@ final _log = DebugTimingLogger('debugger', mute: true);
 /// Responsible for managing the debug state of the app.
 class DebuggerController extends DisposableController
     with AutoDisposeControllerMixin {
-  DebuggerController() {
-    switchToIsolate(serviceManager.isolateManager.selectedIsolate);
+  DebuggerController({bool initialSwitchToIsolate = true}) {
+    if (initialSwitchToIsolate) {
```

Comment:
I added this to prevent the fake from forcing the DebuggerController through isolate subscription. Otherwise, thats a lot of mocking each test needs